src/pages/Thanks/ThanksPage.vue

```diff
@@ -51,56 +51,61 @@
 					</template>
 				</div>
 			</div>
+			<thanks-layout-v2
+				v-if="receipt"
+				:show-mg-cta="!isMonthlyGoodSubscriber && !isGuest && !showAutoDepositUpsell && !hasModernSub"
+				:show-auto-deposit-upsell="!isAutoDepositSubscriber && showAutoDepositUpsell && !hasModernSub"
+				:show-guest-upsell="isGuest"
+				:show-share="loans.length > 0"
+				:thanks-social-share-version="simpleSocialShareVersion"
+				:class="{
+					'tw-mt-4': showAutoDepositUpsell
+				}"
+			>
+				<template #receipt>
+					<checkout-receipt
+						v-if="receipt"
+						:lender="lender"
+						:receipt="receipt"
+					/>
+				</template>
+				<template #ad>
+					<auto-deposit-c-t-a />
+				</template>
+				<template #mg>
+					<monthly-good-c-t-a
+						:headline="ctaHeadline"
+						:body-copy="ctaBodyCopy"
+						:button-text="ctaButtonText"
+					/>
+				</template>
+				<template #share>
+					<social-share
+						v-if="receipt && simpleSocialShareVersion !== 'b'"
+						class="thanks__social-share"
+						:lender="lender"
+						:loans="loans"
+					/>
+					<social-share-v2
+						v-if="receipt && simpleSocialShareVersion === 'b'"
+						class="thanks__social-share"
+						:lender="lender"
+						:loans="loans"
+						:show-header="true"
+					/>
+				</template>
+				<template #guest>
+					<guest-upsell
+						:loans="loans"
+					/>
+				</template>
+			</thanks-layout-v2>
 		</div>
-
-		<thanks-layout-v2
-			v-if="receipt"
-			:show-mg-cta="!isMonthlyGoodSubscriber && !isGuest && !showAutoDepositUpsell && !hasModernSub"
-			:show-auto-deposit-upsell="!isAutoDepositSubscriber && showAutoDepositUpsell && !hasModernSub"
-			:show-guest-upsell="isGuest"
-			:show-share="loans.length > 0"
-			:thanks-social-share-version="simpleSocialShareVersion"
-			:class="{
-				'tw-mt-4': showAutoDepositUpsell
-			}"
-		>
-			<template #receipt>
-				<checkout-receipt
-					v-if="receipt"
-					:lender="lender"
-					:receipt="receipt"
-				/>
-			</template>
-			<template #ad>
-				<auto-deposit-c-t-a />
-			</template>
-			<template #mg>
-				<monthly-good-c-t-a
-					:headline="ctaHeadline"
-					:body-copy="ctaBodyCopy"
-					:button-text="ctaButtonText"
-				/>
-			</template>
-			<template #share>
-				<social-share
-					v-if="receipt && simpleSocialShareVersion !== 'b'"
-					class="thanks__social-share"
-					:lender="lender"
-					:loans="loans"
-				/>
-				<social-share-v2
-					v-if="receipt && simpleSocialShareVersion === 'b'"
-					class="thanks__social-share"
-					:lender="lender"
-					:loans="loans"
-				/>
-			</template>
-			<template #guest>
-				<guest-upsell
-					:loans="loans"
-				/>
-			</template>
-		</thanks-layout-v2>
+		<thanks-page-share v-if="receipt && showNewThanksPage"
+			:lender="lender"
+			:loan="selectedLoan"
+			:simple-social-share-version="simpleSocialShareVersion"
+		/>
```

Comment:
It looks like the receipt prop is missing for this component `:receipt="receipt"`

Reply:
Yes, you are right! 👍